students/hwijeong/westagram/users/views.py

```diff
@@ -1,3 +1,41 @@
+import json
+
 from django.shortcuts import render
+from django.http import JsonResponse
+from django.http.response import HttpResponse
+from django.views import View
+from django.core.exceptions import ValidationError
+from django.db.utils import IntegrityError
+
+from .models import User
+from .validators import isEmailValid, isPasswordValid
+
+class RegisterView(View):
+
+    def post(self, request):   
+        data = json.loads(request.body)
+
+        try:
+            name     = data["name"]
+            email    = data["email"]
+            password = data["password"]
+            contact  = data["contact"]
+            note     = data["note"]
 
-# Create your views here.
+            isEmailValid(email)
+            isPasswordValid(password)
+       
+            user = User.objects.create(
+                name     = name,
+                email    = email,
+                password = password,
+                contact  = contact,
+                note     = note
+            )
+            return JsonResponse({"message": "SUCCESS"}, status=201)
+        except KeyError:
+            return JsonResponse({'message': "Please check the input, all values needed"})
```

Comment:
에러메세지는 그대로 사용자가 보는게 아닌 프론트엔드 개발자에게 보여지는 메세지입니다. 정확한 에러메세지를 리턴해주는게 더 좋을 것 같습니다. 아래 에러도 마찬가지로 수정해주세요.
```suggestion
            return JsonResponse({'message': "KEY_ERROR"})
```